flake-ctl/src/podman.rs

```diff
@@ -74,13 +75,26 @@ pub fn load(oci: &String) -> i32 {
     /*!
     Call podman load with the provided oci tar file
     !*/
+    let mut container_archive: String = oci.to_string();
+    
     info!("Loading OCI image...");
-    info!("podman load -i {}", oci);
-
+    if !Path::new(oci).exists() {
+        let container_archives = oci.to_owned() + "*";
+        // glob puts things in alpha sorted order which is expected to give
+        // us the highest version of the archive
+        for entry in glob(&container_archives)
```

Comment:
Would you like to print a warning for the glob matches not taken into account ?

Reply:
I think we are OK without detailing all the archives we are not using.

Reply:
ok as it prints the one it is using we can also leave it as it is